README.md

```diff
@@ -131,6 +132,44 @@ val exampleBuilder: Example.Builder = exampleType.to(record)
 val copy = exampleType.from(exampleBuilder.build)
 ```
 
+Protobuf works slightly differently in that you need to specify both the type of the case class 
```

Comment:
Every `*Type` works differently, some a lot more fundamentally different than others e.g. `ExampleField` that's not explainable in 2 paragraphs without prior hands on experience.
Also documenting (un)supported features has a very high chance of going out of sync with code.